playwright/_impl/_browser.py

```diff
@@ -50,6 +50,7 @@ def __init__(
         self._is_connected = True
         self._is_closed_or_closing = False
         self._is_remote = False
+        self._is_connected_over_websocket = False
```

Comment:
It seems like you can use is_remote here.

Reply:
remote is also used for connect_over_cdp, we only want to close the underlying connection when connect is used.